styles.css

```diff
@@ -0,0 +1,157 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Rubik", sans-serif;
+  font-size: 62.5%;
+}
+
+.page-footer {
+  background-image: url("https://clickup.com/images/main/bg-main-gradient.svg");
+  width: 100vw;
+}
+
+.flex {
+  display: flex;
+}
+
+.wrapper-page-footer {
```

Comment:
I think one class of either .flex or .wrapper-page-footer is enough. I would simply move display:flex to .wrapper-page-footer

Reply:
display: flex is used only just ones so that's why there is flex class. In html flex used sever time. This saves up some code lines.